ajax/kanban.php

```diff
@@ -190,9 +190,12 @@
     }
 } else if (($_POST['action'] ?? null) === 'move_item') {
     $checkParams(['card', 'column', 'position', 'kanban']);
-    /** @var Kanban|CommonDBTM $kanban */
+    /** @var CommonDBTM $kanban */
```

Comment:
You can remove this comment completely. The `method_exists()` check added on the following lines is the best solution in the 10.0/bugfixes branch as we cannot introduce an interface that would result in a BC-break.
```suggestion
```

Reply:
Ok; lets add some kind of backlog issue to improve this on main then ?

Reply:
Done.